cli/azd/pkg/auth/errors.go

```diff
@@ -62,13 +75,22 @@ func (e *ReLoginRequiredError) init(response *AadErrorResponse, scopes []string,
 		}
 	}
 
+	// The refresh token has expired or is invalid due to sign-in frequency checks by Conditional Access.
 	if slices.Contains(response.ErrorCodes, 70043) {
 		e.scenario = "login expired"
 	}
+
+	// User tried to sign in to a device from a platform not currently supported through Conditional Access policy
+	if slices.Contains((response.ErrorCodes), 50005) {
```

Comment:
Interesting to see the extra surrounding `(` `)` characters here. Usually this would force order of operations, but here I'm wondering if this is a literal that is unnecessary.